README.md

```diff
@@ -55,7 +55,6 @@
 - [Captain hook](https://github.com/stevenpersia/captain-hook) Modest list of hooks.
 - [crooks](https://github.com/chrisjpatty/crooks) A collection of unique React Hooks.
 - [hooks-by-example](https://github.com/latviancoder/hooks-by-example) Collection of beginner-friendly real world examples of hook usage.
-- [Hooks.guide](https://www.hooks.guide/) Collection of React hooks curated by the community.
```

Comment:
The project is still live under https://hooks-guide.netlify.app. Can you update the PR changing the link?

Reply:
Done.